runtime/compiler/ilgen/Walker.cpp

```diff
@@ -4955,7 +4948,17 @@ break
        return NULL;
        }
 
-    int32_t constToLoad = 0;
+   int32_t constToLoad = 0;
+   if(symbol->getRecognizedMethod() == TR::com_ibm_jit_JITHelpers_supportsIntrinsicCaseConversion)
+      {
+      if (cg()->getSupportsInlineStringCaseConversion())
+         constToLoad = 1;
+      else
+         constToLoad  = 0; // should already be 0 but just in case..
+      pop(); //pop up the receiver since it's not used
+      loadConstant(TR::iconst, constToLoad);
```

Comment:
How about `constToLoad = cg()->getSupportsInlineStringCaseConversion() ? 1 : 0` or alternatively just inline the that into `loadConstant`?

Reply:
changed according to the comment. thanks!